client/src/mdn.scss

```diff
@@ -27,6 +32,60 @@ div.sidebar {
       margin-top: 20px;
     }
   }
+
+  details {
+    margin-bottom: 10px;
+    margin-left: 0;
+    padding-left: 20px;
+
+    summary {
+      color: #333;
+      display: inline-block;
+      margin-right: 5px;
+    }
+
+    summary::-webkit-details-marker {
+      display:none;
+    }
+
+
+    summary:before {
+      content: "▶︎";
+      display: inline-block;
+      width: 20px;
+      color: #3d7e9a;
+    }
+  }
+
+  details[open]>summary:before {
+    content: '\25BC\FE0E';
+  }
+
+  ol {
+    list-style-type: none;
+    padding-left: 20px;
+
+    li {
+      padding-left: 0;
+      margin-top: 0;
+      line-height: 1.2;
+      padding-bottom: 10px;
+
+      a {
+        color: #3d7e9a;
```

Comment:
Before it's too late and this minimal SCSS file becomes hard to work on, can you turn these colors and stuff into variables?